src/lein_tools_deps/plugin.clj

```diff
@@ -61,10 +62,3 @@
          resolve-deps
          (merge project))
     project))
-
-(comment
-  (resolve-deps (canonicalise-dep-refs [:system :home "example/deps.edn"]))
```

Comment:
One other thing it'd be good to reinstate this comment, as it's basically a working example usage.

There are some tickets open to improve this through CI/testing etc; but even then it's useful to have this around for repl usage.

Reply:
I like to put my examples in scratch files where I can easily copy them to the REPL.
Speaking of which, where are my scratch files?